static/js/utils/utils.js

```diff
@@ -227,7 +232,7 @@ function updateLabelDiffTime() {
 
         if (disappearsAt.ttime < disappearsAt.now) {
             timestring = 'expired'
```

Comment:
Is negative time even a problem? It seems like we show "expired" in case the time is negative :)

Reply:
> Is negative time even a problem? It seems like we show "expired" in case the time is negative :)

Well, since there is no "ttime" in that object "expired" is never shown.  I actually prefer to see negative times compared to a generic "expired".  But, I'm sure everyone will have their own opinion on this.

I have created some code that you can test over on the w3schools website here:
https://www.w3schools.com/js/tryit.asp?filename=tryjs_editor

If you run my example code, you will see that it produces correct output.  You can play around with the functions to see if any changes you make will result in what you want to see.  Here is the example code:
`<!DOCTYPE html>
<html>
<body>

<p>Play around with the code and click on the "Run" button to view the result.</p>

<p id="demo"></p>

<script>
function getTimeUntil(time, now) {
    // var now = Date.now()
    var tdiff = time - now

    var negTimeUntil = 0
    if (tdiff < 0) {
        negTimeUntil = 1
        tdiff *= -1
        tdiff += 1000
    }

    var sec = Math.floor((tdiff / 1000) % 60)
    var min = Math.floor((tdiff / 1000 / 60) % 60)
    var hour = Math.floor((tdiff / (1000 * 60 * 60)) % 24)

    if (negTimeUntil === 1) {
      tdiff *= -1
      if (hour > 0) hour *= -1
      else if (min > 0) min *= -1
      else sec *= -1
    }

    return {
        total: tdiff,
        hour: hour,
        min: min,
        sec: sec,
        now: now,
        time: time
    }
}

function lpad(str, len, padstr) {
    return Array(Math.max(len - String(str).length + 1, 0)).join(padstr) + str
}

function updateLabelDiffTime(disappearsAt) {

        var hours = disappearsAt.hour
        var minutes = disappearsAt.min
        var seconds = disappearsAt.sec
        var timestring = ''

        if (disappearsAt.ttime < disappearsAt.now) {
            timestring = 'expired'
        } else if (hours != 0) {
            timestring = lpad(hours, 2, 0) + 'h' + lpad(minutes, 2, 0) + 'm' + lpad(seconds, 2, 0) + 's'
        } else {
            timestring = lpad(minutes, 2, 0) + 'm' + lpad(seconds, 2, 0) + 's'
        }

        return timestring
}

timeHatch = 1632659869000
timeExpire = 1632662569000
nowList = [1632659866421, 1632659867421, 1632659868421, 1632659869421, 1632659870421,
           1632659926421, 1632659927421, 1632659928421, 1632659929421, 1632659930421]

var timeExample = ""

for (var i = 0; i < nowList.length; i++) {
  timeUntilStart = updateLabelDiffTime(getTimeUntil(timeHatch, nowList[i]))
  timeUntilExpire = updateLabelDiffTime(getTimeUntil(timeExpire, nowList[i]))
  timeExample = timeExample + 'Start: ' + timeUntilStart + ', Expire: ' + timeUntilExpire + '<br />'
  if (i === 4)
    timeExample = timeExample + '<br /><br />'
}

document.getElementById("demo").innerHTML = timeExample

</script>

</body>
</html>
`

Reply:
Right, ttime was a typo actually. I prefer expired instead of negative time, so I'll close this PR.